entrypoint.sh

```diff
@@ -4,4 +4,12 @@
 
 /usr/bin/openssl req -x509 -newkey rsa:4096 -keyout /etc/jupyterhub/conf/key.pem -out /etc/jupyterhub/conf/cert.pem -days 3650 -nodes -subj '/CN=jupyter'
 
+echo "Adding ${USER}"
+adduser -h "/home/${USER}" -D ${USER}
+mkdir -p /home/${USER}/.sparkmagic
+cp /etc/jupyterhub/conf/sparkmagic/template_sparkmagic_config.json /home/${USER}/.sparkmagic/template_sparkmagic_config.json
```

Comment:
Can we put these template files / directories in /etc/skel during the docker build, then they will automatically land into the user directory on creation, this feels a bit cleaner.

Reply:
Good shout, will do that